src/react_native/core.cljs

```diff
@@ -226,7 +226,3 @@
 (defn open-url [link] (.openURL ^js linking link))
 
 (def set-status-bar-style react-native/StatusBar.setBarStyle)
-
-(defn sharing
-  [content]
-  (.share (.-Share ^js react-native) (clj->js content)))
```

Comment:
Removing this function since we want to use `react-native-share` package and this function is now unused.

Reply:
will cause less confusion this way :+1: